requirements-dev.txt

```diff
@@ -8,3 +8,4 @@ docutils
 pygments >= 2.8
 sphinx
 pydantic < 2
+telnetlib-313-and-up
```

Comment:
Should there be markers to include this only w/ Python 3.13+ or does the module itself take care of that?

Newline missing at the end of the file.

Reply:
Oh yes!  Updated in 9820066c540be594a5f65253187164e0ef0e05ea